### PR TITLE
refactor: 💡 Manage dropdown HDS refactoring for managed-groups

### PR DESCRIPTION
**Pull request by CaptFalc**
Updated the manage dropdown in the managed groups section to reflect HDS design

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14603

## Screenshots (if appropriate)
**Before**
mangroups before
mangroups before2

**After**
mangroups after1
mangroups after2

## How to Test
1. Start up server/Boundary client
2. Navigate to Auth Methods on sidebar
3. Select a resource with type _**LIDC**_ or _**OIDC**_
4. Select managed group from sidebar and select a resource
5. HDS Design should be reflected. 

## Checklist

- [X] I have added before and after screenshots for UI changes
~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

File: ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs [hunk 1]
```diff
@@ -4,17 +4,12 @@
...
 {{#if (can 'delete model' @model)}}
-  <Rose::Dropdown
-    @text={{t 'actions.manage'}}
-    @dropdownRight={{true}}
-    as |dropdown|
-  >
-    <dropdown.button
-      @style='danger'
-      @disabled={{@model.canSave}}
+  <Hds::Dropdown data-test-manage-managedgroups as |dd|>
```
Comment (cameronperera) [inline ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs:7]:
```suggestion
  <Hds::Dropdown data-test-manage-managed-group-dropdown as |dd|>
```
I think we have been adding `-dropdown` to these test selectors.

Comment (lisbet-alvarez) [inline ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs:7]:
```suggestion
  <Hds::Dropdown data-test-manage-managed-groups-dropdown as |dd|>
```

**Review by cameronperera — CHANGES_REQUESTED** [review]
Looks good. Small test selector change.

File: ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs (hunk 1 above)
Comment (CaptFalc) [reply ui/admin/app/components/auth-methods/auth-method/managed-groups/managed-group/actions/index.hbs:7]: I wondered why that looked so off lol

**Review by lisbet-alvarez — APPROVED** [review]
looks great!

**Review by cameronperera — APPROVED** [review]

**Pull request merged**